### PR TITLE
Finished tasks for assignment 1

### PR DESCRIPTION
**Pull request by LucyferYX**

### CODE REVIEW

File: CMP1903_A1_2324/Die.cs [hunk 1]
```diff
@@ -4,20 +4,22 @@
...
-        //Property
-
-
-        //Method
+namespace CMP1903_A1_2324 {
+    internal class Die {
+        // Property
+        public int DieValue {
+            get; // Public by default
+            set; // Private by default
```
Comment (Maxine770) [inline CMP1903_A1_2324/Die.cs]: This isn't private, a potential fix would be changing line 12 to:
private set;

**Review by Maxine770 — COMMENTED** [review]
Met the requirements.
Private properties should be prefixed with an underscore as per the C# google style guide
Named constants preferable to magic numbers as per the C# google style guide
Code is readable and commented when necessary.
Errors handled correctly, unexpected user input exits the program instead of crashing.
Getters and setters by default inherit the access modifier of their property, a line comment with a potential fix has been posted.

File: CMP1903_A1_2324/Die.cs [hunk 2]
```diff
@@ -4,20 +4,22 @@
...
+        }
 
+        // Generates random number
+        private static Random random = new Random();
```
Comment (Maxine770) [inline CMP1903_A1_2324/Die.cs]: Random instance can be made read-only.

**Review by ImSighted — COMMENTED** [review]
Correctly implemented minimum requirements with continuous rolling as an additional feature. 
Effectively handled user input.
Could add addional statistics but isn't crucial.
Naming conventions adhere to the C# google style guide effectively.
Very concise and readable code with effective use of comments.
Basic errors controlled well with the use of "Debug.Assert" with addional context for the user.

**Pull request merged**